build/tutorials/tutorials-contest/drizzle-auction-manager/README.md

```diff
@@ -0,0 +1,970 @@
+
+// Account credentials from which our contract will be deployed
+const MNEMONIC = process.env.MNEMONIC;
+
+module.exports = {
+  contracts_build_directory: "./src/build/contracts",
+  networks: {
+    development: {
+      host: "127.0.0.1",
+      port: 7545,
+      network_id: "*",
+    },
+    fuji: {
+      provider: function () {
+        return new HDWalletProvider({
+          mnemonic: MNEMONIC,
+          providerOrUrl: `https://api.avax-test.network/ext/bc/C/rpc`,
+        });
+      },
+      network_id: "*",
+      gas: 3000000,
+      gasPrice: 470000000000,
+      skipDryRun: true,
+    },
+  },
+  compilers: {
+    solc: {
+      version: "0.8.0"
+    }
+  }
+};
+```
+Here we are setting the `gas` and `gasprice` to an appropriate value of the Avalanche C-chain. The developer may observe that `contract_build_directory` is being used to change the default location of `artifacts` from the project root directory to the `src` folder. Since React is unable to access the files present outside the `src` folder.
+
+## Recieve Avalanche Credentials
+
+For deploying smart contracts we need two things: A node connected to the Avalanche network and an account with few AVAX. Avalanche connected node through RPC (Remote Procedure Call) is provided for free by the Avalanche Networks.
+
+Now we need an Avalanche wallet, where we would keep our funds, required for all the transactions on the network. So, visit [here](https://wallet.avax.network) and create an account. Save the mnemonic in a secure place (we would need it later). Instructions to add funds will be provided later in the tutorial.
+
+### **Add .env file**
+
+Now we need an **Avalanche** wallet, where we would keep our funds, required for all the transactions on the network. Visit the [Avalanche Wallet](https://wallet.avax.network/) and create an account. While creating your account you will see your public address and mnemonic. This public address will be required to transfer funds. Save the **mnemonic** in a secure place, we will need it later. Instructions to add funds will be provided later in the tutorial.
+
+Create a `.env` file in the project root folder. Please take a note that dot (.) is necessary for the `.env` file name. Now copy your Avalanche wallet's mnemonic in the .env file as shown below. In the .env file, **MNEMONIC** should be enclosed within double-quotes (" ").
+
+```bash
+MNEMONIC="<avalanche-wallet-mnemonic>"
+```
+
+> Never share or commit your `.env` file. It contains your credentials like `mnemonics`. Therefore, it is advised to add `.env` to your `.gitignore` file.
+
+Now, the project setup is completed, and run the command given below in the project root folder to check whether we are on the same page or not.
+
+```bash
+npm start
+```
+
+## Create Auction smart contract
+
+Create an `Auction.sol` (sol stands for solidity) file inside the contracts directory and paste the following code given below:
+
+```javascript
```

Comment:
Could you please use the correct language formatting? So `solidity` here instead of `javascript`?